api/api.go

```diff
@@ -9,18 +10,23 @@ import (
 type API struct {
 	db        *database.ServiceDB
 	container *container.Container
+	cfg       *config.Config
```

Comment:
There is already issue https://github.com/mesg-foundation/core/issues/495 that you've created, about not stroing config inside [Container](https://github.com/mesg-foundation/core/blob/dev/container/container.go#L22).

It should be passed as args/option to New func.

Reply:
yes, it is better to not access config inside container package. in this case we access it from api package.

actually i'd rather doing all this initilazitions in the core package and propagate it down to initialization of _grpc server_ & _api_. this way we can reuse everything.